### ISSUE TITLE
Hosted embed JS error

### ISSUE THREAD

**Issue by bradders** [usefathom/fathom#124]
Assuming issues regarding the hosted solution can live here too? Please nuke if not.

Adding the snippet to my site (avalog.co) throws this JavaScript error:

`TypeError: d.get is not a function ... tracker.js:1:1760`

I've removed the tracker from the site, as it nukes my site JS, but it's only a prototype/side-project so let me know if seeing the issue live helps and I'll throw it back on the site.

Attached is a screenshot of the trace, if that helps.
<img width="1308" alt="screenshot 2018-09-13 09 45 16" src="https://user-images.githubusercontent.com/840983/45503397-970d8800-b73b-11e8-9f93-06877ee8f146.png">

**Comment by dannyvankooten** [comment]
Hey @bradders,

Sure thing! Does the site in question load Dojo framework by any chance, that you know of? It seems something is overriding the `require` function that is defined by the Browserify bundle and Dojo is known to do that.

Would it be possible for you to share a link to the site you're adding the tracking script to (you can do this privately by emailing us, support@usefathom.com). That'd be super useful in finding the culprit!

**Comment by bradders** [comment]
The JS portion of the site is definitely in need of updating, and requireJS will likely get replaced in the future, but it would be sweet if fathom could live alongside requireJS for the time being.

**Comment by dannyvankooten** [comment]
Yeah it should absolutely be able to co-exist. We'll get that failsafed!

**Comment by dannyvankooten** [comment]
Hey @bradders,

b75506ce8f5d53308b8385ae656e7fd6c248e8fa should prevent this from occurring. Your Fathom instance is already updated to the latest development release, so if you try adding the snippet again now it should work without any issues.

Thanks for catching this!

**Comment by bradders** [comment]
Thanks, she's working like a charm!
